docs/source/install-plone-restapi.rst

```diff
@@ -0,0 +1,62 @@
+Getting started with plone.restapi
+==================================
+If you haven't installed Plone locally, proceed to the next section, that is, installing Plone with plone.restapi. Make sure you have Python 2.7 and https://virtualenv.pypa.io installed.
```

Comment:
Instead of directly linking `https://virtualenv.pypa.io` I'd recommend:
```
virtualenv_
// content here
.. links
.. _virtualenv: https://virtualenv.pypa.io
```